packages/fxa-content-server/app/scripts/app.js

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import 'babel-polyfill';
```

Comment:
This is deprecated.  Ref: https://babeljs.io/docs/en/babel-polyfill

Reply:
I know we've intentionally avoided babel-polyfill in the past because it adds a significant weight to the page. What specifically needed this added? We should avoid polyfills when we can.

Reply:
Was able to reduce this down to just `regenerator-runtime/runtime` which we need for some of the async functions